contracts/LockedPoolz.sol

```diff
@@ -14,9 +14,10 @@ contract LockedPoolz is LockedManageable {
         uint256 _NewAmount,
         address _NewOwner
     ) internal returns (uint256 newPoolId) {
-        uint leftAmount = remainingAmount(_PoolId);
+        uint256 leftAmount = remainingAmount(_PoolId);
         require(leftAmount > 0, "Pool is Empty");
         require(leftAmount >= _NewAmount, "Not Enough Amount Balance");
+        require(_NewAmount * 10**18 >= _NewAmount, "Invalid amount");
```

Comment:
@YouStillAlive please add a revert test to trigger this error. Probably this error msg won't be thrown because safemath will handle it.

Reply:
lets try:
require(_NewAmount * 10**18 > _NewAmount, "Invalid amount");